src/woo_publications/publications/api/filters.py

```diff
@@ -175,6 +180,27 @@ class PublicationFilterSet(FilterSet):
         field_name="registratiedatum",
         lookup_expr="lte",
     )
+    archiefactiedatum_vanaf = filters.DateTimeFilter(
+        help_text=_(
+            "Filter publications that are gonna be archived/disposed after or on the given value."
```

Comment:
```suggestion
            "Filter publications where the archive action date is after or on the given value."
```

& update the other help texts too please